### DISCUSSION TITLE
182

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#152] [General]
# 潮流周刊第182期 - 煎个牛排

封面图拍摄来源周末自己做的牛排，先用黑胡椒、盐、油抹匀腌制 5min，然后加油一边煎一分钟，然后小火加点儿黄油、大蒜、小西红柿、迷迭香，就可以摆盘开吃了，味道 YYDS。

https://weekly.tw93.fun/posts/182-%E7%85%8E%E4%B8%AA%E7%89%9B%E6%8E%92

**Comment by 11ze** [comment]
可以试试微信输入法的常用语

**Comment by zendext** [comment]
Rime 的自定义短语也不错
